sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -183,6 +183,14 @@ public enum AccountKind
         BlobStorage = 1,
         StorageV2 = 2,
     }
+    public partial class AppendBlobOpenWriteOptions
+    {
+        public AppendBlobOpenWriteOptions() { }
+        public int BufferSize { get { throw null; } set { } }
+        public Azure.Storage.Blobs.Models.AppendBlobRequestConditions Conditions { get { throw null; } set { } }
+        public bool Overwrite { get { throw null; } set { } }
```

Comment:
I have the overwrite parameter because it is present in V11.  https://github.com/Azure/azure-storage-net/blob/master/Lib/ClassLibraryCommon/Blob/CloudAppendBlob.cs#L54.  Same for PageBlob and BlockBlob.  I'm not opposed to getting rid of it.